src/components/FlippableHandler.js

```diff
@@ -32,21 +33,60 @@ class FlippableHandler {
                         window.setTimeout(() => $oldEl.classList.remove('display-flex'), 300);
                         window.setTimeout(() => {
                             $rotationContainer.classList.toggle('flipped', $newEl.classList.contains(classname));
+                            FlippableHandler._updateContainerHeight($newEl);
                         }, 0);
+                    } else {
+                        FlippableHandler._updateContainerHeight($newEl || undefined);
                     }
                 } else if (newHash) {
                     const $newEl = document.querySelector(newHash);
                     if ($newEl && $newEl.classList.contains(classname)) {
                         $rotationContainer.classList.add('disable-transition');
+                        FlippableHandler._updateContainerHeight($newEl);
                         window.setTimeout(() => {
                             $rotationContainer.classList.toggle('flipped', $newEl.classList.contains(classname));
                             window.setTimeout(() => $rotationContainer.classList.remove('disable-transition'), 10);
                         }, 0);
+                    } else {
+                        window.setTimeout(() => {
+                            $rotationContainer.classList.add('disable-transition');
```

Comment:
The first timeout is not needed anymore, you're right.
But the second timeout as well as setting / unsetting `disable-transition` is needed to avoid animating the height change at the loading of the page.